src/main/java/com/spikes2212/command/drivetrains/swerve/commands/TurnModules.java

```diff
@@ -0,0 +1,119 @@
+
+        this.frontLeftModule = drivetrain.getFrontLeftModule();
+        this.frontRightModule = drivetrain.getFrontRightModule();
+        this.backLeftModule = drivetrain.getBackLeftModule();
+        this.backRightModule = drivetrain.getBackRightModule();
+    }
+
+    /**
+     * Constructs a new {@link TurnModules} command that moves the given {@link SwerveDrivetrain}
+     * {@link SwerveModule}s to a certain degree.
+     *
+     * @param drivetrain             the swerve drivetrain this command operates on
+     * @param frontLeftDesiredAngle  the desired front left degree
+     * @param frontRightDesiredAngle the desired front right degree
+     * @param backLeftDesiredAngle   the desired back left degree
+     * @param backRightDesiredAngle  the desired back right degree
+     */
+    public TurnModules(SwerveDrivetrain drivetrain, double frontLeftDesiredAngle, double frontRightDesiredAngle,
+                       double backLeftDesiredAngle, double backRightDesiredAngle) {
+        this(drivetrain, Rotation2d.fromDegrees(frontLeftDesiredAngle), Rotation2d.fromDegrees(frontRightDesiredAngle),
+                Rotation2d.fromDegrees(backLeftDesiredAngle), Rotation2d.fromDegrees(backRightDesiredAngle));
+
+    }
+
+    /**
+     * Moves each {@link SwerveModule} in the given {@link SwerveDrivetrain}
+     * to the desired angle in degrees.
+     */
+    @Override
+    public void execute() {
+        frontLeftModule.setTargetAngle(frontLeftDesiredAngle);
+        frontRightModule.setTargetAngle(frontRightDesiredAngle);
+        backLeftModule.setTargetAngle(backLeftDesiredAngle);
+        backRightModule.setTargetAngle(backRightDesiredAngle);
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (frontLeftModule.getTurnMotor().onTarget(
+                UnifiedControlMode.POSITION,
+                frontLeftModule.getTurnMotorPIDSettings().getTolerance(),
+                frontLeftDesiredAngle.getDegrees()) &&
+                frontRightModule.getTurnMotor().onTarget(
+                        UnifiedControlMode.POSITION,
+                        frontRightModule.getTurnMotorPIDSettings().getTolerance(),
+                        frontRightDesiredAngle.getDegrees()) &&
+                backLeftModule.getTurnMotor().onTarget(
+                        UnifiedControlMode.POSITION,
+                        backLeftModule.getTurnMotorPIDSettings().getTolerance(),
+                        backLeftDesiredAngle.getDegrees()) &&
+                backRightModule.getTurnMotor().onTarget(
+                        UnifiedControlMode.POSITION,
+                        backRightModule.getTurnMotorPIDSettings().getTolerance(),
+                        backRightDesiredAngle.getDegrees())) {
+            lastTimeNotOnTarget = Timer.getFPGATimestamp();
+        }
+
+        return Timer.getFPGATimestamp() -
+                lastTimeNotOnTarget >= frontLeftModule.getTurnMotorPIDSettings().getWaitTime();
```

Comment:
Kind of weird but I get it. If you want to, you can add an optional `waitTime` parameter to the command.